src/runtime/manifest-parser.pegjs

```diff
@@ -1472,15 +1472,21 @@ RecipeSlot
   }
 
 SchemaInline
-  = names:((upperIdent / '*') whiteSpace?)* '{' multiLineSpace fields:(SchemaInlineField (',' multiLineSpace SchemaInlineField)*)? ','? multiLineSpace '}'
+  = names:((upperIdent / '*') whiteSpace?)* fields:SchemaInlineFields
   {
     return toAstNode<AstNode.SchemaInline>({
       kind: 'schema-inline',
       names: optional(names, names => names.map(name => name[0]).filter(name => name !== '*'), ['*']),
-      fields: optional(fields, fields => [fields[0], ...fields[1].map(tail => tail[2])], []),
+      fields
     });
   }
 
+SchemaInlineFields
+  = '{' multiLineSpace fields:(SchemaInlineField (',' multiLineSpace SchemaInlineField)*)? ','? multiLineSpace '}'
```

Comment:
If you wanted, you could use some helpers I defined recently: `openBrace`, `commaOrNewline`, `closeBrace`. They play nicely together to let you parse things like:

```
{a, b, c}

{a, b, c,}

{
  a,
  b,
  c,
}

{
  a
  b
  c
}
```

You might not want to accept newlines in lieu of commas though..